easybuild/easyconfigs/m/MultiQC/MultiQC-0.9-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Adam Huffman
+# The Francis Crick Institute
+easyblock = 'PythonPackage'
```

Comment:
@verdurin They'll get installed automatically even if they're not included (see EB log), but then we don't have control over which versions are being used for them...